subset/bacnet/bacnetTests/src/main/java/helper/FileManager.java

```diff
@@ -0,0 +1,64 @@
+package helper;
+
+import java.io.File;
+
+public class FileManager {
+
+    private String filePath = "";
+    private String csvName = "pics";
+    private String csvExtension = ".csv";
+    private boolean debug = false;
+
+    public boolean checkDevicePicCSV() {
+        String csvFolder = getCSVPath();
+        try{
+            File[] listFiles = new File(csvFolder).listFiles();
+            for (int i = 0; i < listFiles.length; i++) {
+                if (listFiles[i].isFile()) {
+                    String fileName = listFiles[i].getName();
+                    if (fileName.contains(csvName)
+                            && fileName.endsWith(csvExtension)) {
+                        System.out.println("pics.csv file found in /config/type.");
+                        setFilePath(fileName);
+                        return true;
+                    }
+                }
+            }
+            String errorMessage = "Pics.csv not found.\n";
```

Comment:
@JoelRedstone The file path is also useful if it's printed when it's _not_ found